migration/src/db/postgres.ts

```diff
@@ -1,24 +1,23 @@
-import config from "config";
+import { env } from "src/env";
 import { DataSource } from "typeorm";
 import { CourseEntity } from "../entity/Course";
 import { ReviewEntity } from "../entity/Review";
 import { UserEntity } from "../entity/User";
-import { IDatabaseConfig } from "src/types/IConfig";
 
 export default class Postgres {
   private logger = console;
   private dbConnection: DataSource;
 
   constructor(readonly connectionName: string) {
-    const dbConfig: IDatabaseConfig = config.get("database");
     this.dbConnection = new DataSource({
+      type: "postgres",
```

Comment:
put this in env too, but make default postgres

Reply:
the datasource class wants us to pass in the string literal 'postgres' and won't accept the envsafe value 😭 i tried messing with envsafe but it wont let me narrow the condition for the type down to fit here.

Reply:
`DB_TYPE: str({ choices: ["postgres"], devDefault: "postgres" })`

Reply:
yup, i tried that but the type check still wont let it be :((

Reply:
i added the haram type reinforcements